mindocr/postprocess/det_fce_postprocess.py

```diff
@@ -295,66 +301,36 @@ def __init__(
         self.beta = beta
         self.box_type = box_type
 
-        # def __call__(self, preds, shape_list):
-        #     score_maps = preds['maps']
-        #     return self.get_boundary(score_maps, shape_list)
-
-    def __call__(self, preds, meta_info, **kwargs):
-        shape_list = meta_info[0].numpy()
+    def _postprocess(self, pred, **kwargs):
```

Comment:
Does switching from your `FCEPostprocess`'s `__call__` to `_postprocess` affect the prediction score? Because `DetBasePostprocess`'s `__call__` can be slightly different.

Reply:
I have transferred all the code in the original `__call__` to `_postprocess`